theory-test-admission-dao/integration-test/AdmissionDAO.integration.js

```diff
@@ -0,0 +1,34 @@
+require('./AWSConfig');
```

Comment:
The AWSConfig is still needed for local dev purpose, so it is best to include it in the src dir rather than the test dir. And all we want is to exclude it from the build for production. Please take a look at what I did in the candidate app.